common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/setup_venv.sh

```diff
@@ -24,16 +24,10 @@ source "${VENV_DIR}/bin/activate" || echo "Could not activate venv"
 # Upgrade pip and install requirements. 'python' is used here in order to
 # reference to the python executable from the venv.
 python -m pip install --upgrade pip || echo "Could not upgrade pip"
-python -m pip install iree-tools-tf -f https://openxla.github.io/iree/pip-release-links.html
 
-# Run through all model directories and install requirements.
-TF_MODELS_DIR="$(dirname $(dirname $(dirname "${TD}")))/models/tf"
-find "${TF_MODELS_DIR}" -type d | while read dir; do
-  if [[ -f "${dir}/requirements.txt" ]]; then
```

Comment:
Maybe we can still keep this loop but remove tf-nightly, pillow, keras, and transformers from those `requirements.txt`? (or remove their `requirements.txt` if they are empty) Just in case those models need additional packages

Reply:
Added some code to remove `tensorflow` from the requirements.txt file so that it doesn't interfere with `tf-nightly`.